### ISSUE TITLE
Important Tools Bar Disapper from Google Chrom View

### ISSUE THREAD

**Issue by PTan2529** [jgraph/drawio#3061]
### Preflight Checklist

* [X ] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [ X] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

You must agree to search and the code of conduct. You must fill in this entire template. If you delete part/all or miss parts out your issue will be closed.

**Describe the bug**
Tools Bar and Left Side bar are disappear from Goolge Chrome Website.
So I cannot
1. Go to another page on the file
2. Add new Shape
See the Picture  
![image](https://user-images.githubusercontent.com/113820369/190889468-97845c06-cc92-4a33-91f9-bb0e03274334.png)

**To Reproduce**
Just Only Open the file. And those functions are gone. 

**Expected behavior**
It suppose to show up like this 
![image](https://user-images.githubusercontent.com/113820369/190889438-2567a690-b948-488c-9290-cff6c56dd001.png)

**Screenshots**
Done in previous section.

**draw.io version (In the Help->About menu of the draw.io editor):**
- The problem is not on desktop version. It is on Web browser version. Can't even know which version it is. 

**Desktop (please complete the following information):**
 - OS: Windows 10

**Additional context**
- None

**Comment by davidjgraph** [comment]
Any browser extensions?

**Comment by PTan2529** [comment]
> Any browser extensions?

No extension at all. Just use browser alone.

**Comment by PTan2529** [comment]
While I'm using Microsoft Edge -> Everything seems to work normal.
Just only on Google Chrome

**Comment by Franreno** [comment]
This is also happening to me. 

Opening the diagram on Vivaldi Browser (Chromium-based) results in the same problem that @PTan2529 has. 
![image](https://user-images.githubusercontent.com/67326251/190919550-a6de18de-c79a-4916-aaf7-0ff615e56274.png)

However, the problem still happens with other google accounts.

The problem doesn't occur on the Firefox Browser, see the screenshot below. 
![image](https://user-images.githubusercontent.com/67326251/190919732-96b7138f-c305-4b57-be5c-da8601a92fac.png)

I already tried to clean up cookies and cache but the problem persists.

**Comment by Franreno** [comment]
Disabling [Video Speed Controller Extension](https://chrome.google.com/webstore/detail/video-speed-controller/nffaoalbilbmmfgbnbgppjihopabppdk) solved the issue for me!

**Comment by PTan2529** [comment]
> Disabling [Video Speed Controller Extension](https://chrome.google.com/webstore/detail/video-speed-controller/nffaoalbilbmmfgbnbgppjihopabppdk) solved the issue for me!

@Franreno 
That's a great guideline though. 
I have tried closing the Extension. And finally found the main cause of it.
My ClickUP extension make the display error. 
![image](https://user-images.githubusercontent.com/113820369/190921251-081121bc-0720-4d28-94e4-630a5b84f0e0.png)

I think dev team should be noted about this. As I think it is a bug problem. Coz I have used this extension for around 2 years without problem. And somehow, this error show up not only for me. 

Hope the dev notice about this, and fix it up. 

Always support since Draw.io
Cheers

**Comment by davidjgraph** [comment]
@PTan2529 I don't understand your answers. You said "No extension at all." and later that an extension causes this.

**Comment by PTan2529** [comment]
> @PTan2529 I don't understand your answers. You said "No extension at all." and later that an extension causes this.

Sorry my bad. This extension (ClickUP) has been used for 2 years with diagram.net and has no problem. And it just cause the issue last 2 days. Thx

**Comment by davidjgraph** [comment]
Our guess is that these extensions are changing CSS name somewhere. We made a change in 20.3.3 that should avoid this.

**Comment by PTan2529** [comment]
> Our guess is that these extensions are changing CSS name somewhere. We made a change in 20.3.3 that should avoid this.

@davidjgraph the 20.3.3 is already go live? I still have same issue while using ClickUP Extension.

**Comment by davidjgraph** [comment]
It is in production, yes. The version is in the help menu.
